### PR TITLE
Fix windows test failures

### PR DESCRIPTION
**Pull request by Almenon**
# Pull Request

## Related Github Issues

- #300 

## Description
* ~~Fix env var parsing bug~~
* Add personal vscode workspace file to gitignore
* Update readme

### CODE REVIEW

File: cmd/generate.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func getEnvs() map[string]string {
 	m := make(map[string]string)
 
 	for _, env := range envs {
-		results := strings.SplitN(env, "=", 1)
```
Comment (dennislapchenko) [inline cmd/generate.go:34]: a small unittest for this would be nice

Comment (Almenon) [reply cmd/generate.go:34]: We already have a test for this, check out the failing test for main branch

Comment (Almenon) [reply cmd/generate.go:34]: @dennislapchenko following up on this

Comment (pseudomorph) [reply cmd/generate.go:34]: Seems like this should be done with Cut, as per the docs.

https://pkg.go.dev/strings#SplitN

```To split around the first instance of a separator, see Cut.```


Perhaps something like:
```diff
- results := strings.SplitN(env, "=", 2)
- m[results[0]] = results[1]
+ varName, varValue, found := strings.Cut(env, "=")
+   if found {
+     m[varName] = m[varValue]
+   }

Comment (Almenon) [reply cmd/generate.go:34]: either way works, let's just go with what's already in the code and passing tests

**Comment by pseudomorph** [comment]
@Almenon - Any update on this? This Blocks #320

**Comment by Almenon** [comment]
I'm waiting for a review from @dmattia or others, I don't have the ability to self-merge.

**Comment by Almenon** [comment]
It looks like David fixed the env var parsing bug in a commit directly to master, I guess they didn't see this PR.  @pseudomorph if you rebase your tests should work now.

@dmattia can I get a review please? This PR used to fix the env var parsing bug and a windows bug. Now that you've fixed the first bug but this PR just fixes the windows bug, so it's a small baby-rabbit sized PR 🐰 .

**Review by pseudomorph — APPROVED** [review]
Looks good to me.

**Pull request merged**